src/components/memberships/FetchMembershipsForm.js

```diff
@@ -73,8 +87,13 @@ class FetchMembershipsForm extends React.Component {
   }
 }
 
-export default connect(
-  state => ({
-    memberships: state.membeships,
-  })
-)(FetchMembershipsForm);
+function mapStateToProps(state, ownProps) {
```

Comment:
i am not a fan of mixing es5 with es6. therefore you might want to consider rewriting function to consts. The codebase will be much cleaner